packages/core/README.md

```diff
@@ -70,6 +70,48 @@ const governor = await governor.deployed();
 Note: this use case is not particularly common, but it is sometimes useful to have access to multiple abi versions
 side-by-side.
 
+## Deployment with Hardhat
+
+Here is a list of scripts you can execute:
+
+`yarn void:deploy`
+
+This will deploy your contracts on the in-memory hardhat network and exit, leaving no trace. Quickest way to ensure that deployments work as intended without consequences.
```

Comment:
do you want to put the description _above_ the command? makes it a bit easier to read.